internal/pkg/cniutils/cniutils.go

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package cniutils provides helper functions to parse CNI results.
+package cniutils
+
+import (
+	"fmt"
+
+	"github.com/containernetworking/cni/pkg/types/current"
+)
```

Comment:
TODO: add copyright header here

Reply:
which one?

Reply:
I took the code here: https://github.com/firecracker-microvm/firecracker-go-sdk/blob/master/cni/internal/cniutil.go

As it's internal package, I can't import it. So we might need to add copyright notice somewhere after `import` so we don't break the license rules for the original code.

Reply:
ahh, gotcha.